### PR TITLE
Fixed Spark test to run within Framework Tests

### PR DESCRIPTION
**Pull request by susanxhuynh**
(1) Uninstall and janitor in module teardown, because there are other framework tests running after Spark tests on the same cluster.
(2) Some changes related to new HDFS: new endpoint, number of tasks, longer timeout.

### CODE REVIEW

**Comment by susanxhuynh** [comment]
retest this please

**Comment by susanxhuynh** [comment]
retest this please

**Comment by susanxhuynh** [comment]
retest this please

**Comment by susanxhuynh** [comment]
retest this please

File: tests/test.py [hunk 1]
```diff
@@ -51,16 +57,25 @@ def _require_package(pkg_name):
...
 
-DEFAULT_HDFS_TASK_COUNT=8
+DEFAULT_HDFS_TASK_COUNT=10
 def _is_hdfs_ready(expected_tasks = DEFAULT_HDFS_TASK_COUNT):
     running_tasks = [t for t in shakedown.get_service_tasks('hdfs') \
                      if t['state'] == 'TASK_RUNNING']
     return len(running_tasks) >= expected_tasks
 
 
+def _run_janitor(service_name):
```
Comment (mgummelt) [inline tests/test.py]: Is this not already in shakedown?  Wasn't nick putting this in shakedown?  If not, we should.

Comment (susanxhuynh) [inline tests/test.py]: Nick was asked to separate out the janitor code from the rest of his shakedown changes. His janitor code PR is still open: https://github.com/dcos/shakedown/pull/119

**Pull request merged**